src/utils/isNodeProcess.react-native.test.ts

```diff
@@ -33,11 +33,12 @@ beforeAll(() => {
    * @see https://github.com/facebook/react-native/blob/6d6c68c2c639b6473e049f7d916690b92e921c7e/Libraries/Core/setUpNavigator.js
    * @see https://github.com/facebook/react-native/blob/6d6c68c2c639b6473e049f7d916690b92e921c7e/Libraries/Core/setUpGlobals.js
    */
-  Object.defineProperty(global.process, 'env', {
-    get: () => ({
+  // @ts-ignore
```

Comment:
Please, instead of `ts-ignore`, could you describe this in the `declare global` type override at the top of the file?
I understand that most likely you're using `ts-ignore` here to bypass `"process" is readonly", but I believe you can drop the `readonly` modifier form an interface key (we re-declare `interface Global` on top), allowing to write into `global.process`.

Reply:
Sorry, I don't quite understand. I think `re-declare interface Global` will merge `original interface Global declaration` and `global.process` will still be `Process` type? And that will refuse the assignment of `global.process`, which requires many other properties, such as `stdin, stdout` and so on. Please let me know if my thoughts are wrong. Thanks for the comment!

Reply:
Oh, I see. Then I'd recommend to cast the override object `as any`. It's a less harmful override than ignoring the entire line:

```ts
global.proces = { ... } as any
```

What do you think?

Reply:
Thanks for the comment! I have changed the code, `as any` is better.